tutorials/polkadot-sdk/parachains/zero-to-hero/obtain-coretime.md

```diff
@@ -37,7 +37,7 @@ Purchasing bulk coretime involves purchasing a core from the [Coretime Chain](/p
 - [RegionX Coretime Marketplace (includes Paseo support)](https://app.regionx.tech){target=\_blank}
 - [Lastic Coretime Marketplace](https://www.lastic.xyz/polkadot/bulkcore1){target=\_blank}
   
-!!!info "Obtaining a Core on Paseo"
+!!!tip
```

Comment:
There should be a period after the link under this admonition. Sorry, I know it's sort of unrelated, but feel like it's a quick fix